csp-mixin/panels.libsonnet

```diff
@@ -842,5 +842,312 @@ local commonlib = import 'common-lib/common/main.libsonnet';
+      + commonlib.panels.generic.stat.base.new(
+        'VM Availability',
+        [
+          this.signals.virtualMachinesOverview.vmAvailability.asTarget()
+          + g.query.prometheus.withFormat('time_series')
+          + g.query.prometheus.withInstant(true)
+          + g.query.prometheus.withRange(false),
+        ],
+        'Measure of Availability of Virtual machines'
+      )
+      + g.panel.stat.options.withColorMode('background')
+      + g.panel.stat.standardOptions.withUnit('short')
+      + g.panel.stat.standardOptions.withOverrides([
+        {
+          matcher: {
+            id: 'byType',
+            options: 'number',
+          },
+          properties: [
+            {
+              id: 'mappings',
+              value: [
+                {
+                  options: {
+                    '0': {
+                      color: 'red',
+                      index: 1,
+                      text: 'Not Available',
+                    },
+                    '1': {
+                      color: 'green',
+                      index: 0,
+                      text: 'Available',
+                    },
+                  },
+                  type: 'value',
+                },
+              ],
+            },
+          ],
+        },
+      ]),
+
+    _avm_timeSeriesCommon()::
+      g.panel.timeSeries.standardOptions.color.withMode('palette-classic')
+      + g.panel.timeSeries.standardOptions.withNoValue('0')
+      + g.panel.timeSeries.standardOptions.withOverrides([]),
+
+    avm_cpu_utilization:
+      this.signals.virtualMachines.cpuUtilization.asTimeSeries()
+      + commonlib.panels.generic.timeSeries.base.stylize()
+      + g.panel.timeSeries.standardOptions.withUnit('percent')
+      + self._avm_timeSeriesCommon(),
+
+    avm_available_memory:
+      this.signals.virtualMachines.availableMemory.asTimeSeries()
+      + commonlib.panels.generic.timeSeries.base.stylize()
+      + g.panel.timeSeries.standardOptions.withUnit('decbytes')
+      + self._avm_timeSeriesCommon(),
+
+    avm_cpu_credits_consumed:
+      this.signals.virtualMachines.cpuCreditsConsumed.asTimeSeries()
+      + commonlib.panels.generic.timeSeries.base.stylize()
+      + g.panel.timeSeries.standardOptions.withUnit('short')
+      + self._avm_timeSeriesCommon(),
+
+    avm_cpu_credits_remaining:
+      this.signals.virtualMachines.cpuCreditsRemaining.asTimeSeries()
+      + commonlib.panels.generic.timeSeries.base.stylize()
+      + g.panel.timeSeries.standardOptions.withUnit('short')
+      + self._avm_timeSeriesCommon(),
+
+    avm_disk_total_bytes:
+      this.signals.virtualMachinesOverview.diskReadBytes.asTimeSeries()
+      + commonlib.panels.generic.timeSeries.base.stylize()
+      + this.signals.virtualMachinesOverview.diskWriteBytes.asPanelMixin()
+      + g.panel.timeSeries.standardOptions.withOverrides([])
+      + g.panel.timeSeries.standardOptions.withUnit('decbytes')
```

Comment:
You don't need this, as units are automatically applied from signals when asTimeseries() or asPanelMixin() are used.